honeybadgermpc/batch_reconstruction.py

```diff
@@ -104,13 +106,14 @@ def list_transpose(lst):
     return None
 
 
-async def batch_reconstruct(secret_shares, p, t, n, myid, send, recv, config=None,
-                            use_fft=False, debug=False):
+async def batch_reconstruct(secret_shares, p, t, n, f, myid, send, recv, config=None,
+                            use_fft=False, debug=False, wait_for_all=False):
     """
     args:
       shared_secrets: an array of points representing shared secrets S1 - SB
       p: field modulus
       t: degree t polynomial
+      f: number of faults to tolerate
       n: total number of nodes n >= 3t+1
```

Comment:
this comment should now be.... `n >= t + 2f+1` for robust decoding, `n >= t + f + 1` for malicious secure decoding

Reply:
What is malicious secure decoding?

Reply:
it guarantees it will not output a *wrong* answer if `f` nodes are malicious, but it may also fail to produce any answer